### ISSUE TITLE
[feature]: Can I have my collection sorted alphabetically?

### ISSUE THREAD

**Issue by klesh** [hoppscotch/hoppscotch#2261]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

![image](https://user-images.githubusercontent.com/61080/163168378-a776efb5-ed36-4c70-807f-5eb903d1eb91.png)

### Why should this be worked on?

It is hard for me to find particular entry that I need, I wish they could be sorted alphabetically.

**Comment by TeamPIDU** [comment]
Yes please add Alphabetical Sorting or Drag Drop Sort - so as Request list can be managed properly

**Comment by AbderrahmaneGuermat** [comment]
I'll do something like that

**Comment by jamesgeorge007** [comment]
Hi, closing this issue since the ability to sort collections is now available in the latest release. Please feel free to let us know if you have any feedback.
